src/main/kotlin/domain/CellState.kt

```diff
@@ -0,0 +1,11 @@
+package domain
+
+data class CellState(private val open: Boolean = false, private val mine: Boolean = false) {
+    fun isOpen(): Boolean = open
+
+    fun isMine(): Boolean = mine
+
+    fun open(): CellState = CellState(true, mine)
+
+    fun layMine(): CellState = CellState(open, true)
```

Comment:
```suggestion
    fun open(): CellState = CellState(open = true, mine = mine)

    fun layMine(): CellState = CellState(open = open, mine = true)
```
함수 파라미터를 봤을 때 어떤 것인지 잘 보이지 않는다면, argument name을 활용하면 가독성이 올라가요